src/components/pages/Publications/PublicationsCard.tsx

```diff
@@ -9,6 +9,19 @@ import {PublicationProps} from "./PublicationsConstants"
 
 export function PublicationsCard(props: PublicationProps) {
   const [state] = useContext(AppContext);
+
+  const dateFormatsByLanguage = {

```

Comment:
Slightly cleaner pattern would be to make a shared utility function to perform this logic (you could use it both in the publicationsCard and publicationsItem components). It could have the following signature:

```
function getFormattedDate(day, month, year, language)
```